cli/pkg/utils/kubectl.go

```diff
@@ -22,5 +24,26 @@ func Kubectl(stdin io.Reader, args ...string) error {
 	}
 	kubectl.Stdout = os.Stdout
 	kubectl.Stderr = os.Stderr
+	kubectl.Env = os.Environ()
 	return kubectl.Run()
 }
+
+func KubectlOut(stdin io.Reader, args ...string) (string, error) {
+	kubectl := exec.Command("kubectl", args...)
+	if stdin != nil {
+		kubectl.Stdin = stdin
+	}
+
+	out := &bytes.Buffer{}
+
+	kubectl.Stdout = out
+	kubectl.Stderr = out
+	kubectl.Env = os.Environ()
+	err := kubectl.Run()
+
+	if err != nil {
+		return "", errors.Wrap(err, out.String())
+	}
+
+	return out.String(), nil
+}
```

Comment:
Where is this function being used?

Reply:
right now only in wasme but i think it makes sense with these other functions. eventually i want to move these to `go-utils`

`go-utils` have a parallel for this in `testutils` specifically designed for Ginkgo tests.

Reply:
If it's only in wasme can we move it there rather than in autopilot?

Reply:
i'd rather not since i would like to move these utils as a group to a standard utils package, either in autopilot, go-utils, or another location. for now i don't think it makes sense to put `Kubectl` and `KubectlOut` in separate packages